app/src/main/java/com/example/androidthings/assistant/AssistantActivity.java

```diff
@@ -173,12 +180,23 @@ public void onSpeechRecognition(List<SpeechRecognitionResult> results) {
                     }
                 })
                 .setConversationCallback(new ConversationCallback() {
+                    @Override
+                    public void onResponseStarted() {
+                        if (mDac != null) {
+                            try {
+                                mDac.setSdMode(Max98357A.SD_MODE_LEFT);
```

Comment:
can you also add a call to `mDac.setSdMode(Max98357A.SD_MODE_SHUTDOWN)` in `onConversationFinished`

Reply:
`onResponseFinished`